devices/meamor_3way-switch.yaml

```diff
@@ -0,0 +1,100 @@
+esphome:
+  name: meamor_3switch
+  platform: ESP8266
+  board: esp01_1m
+#The Meamor is identical to the Sonoff UK 3, but can handle 10 Amps. 
+#You can find it on Amazon.de searching for "Meamor Lichtschalter"
+
+wifi:
+  ssid: 'MySSID'
+  password: 'MyPassword
+    static_ip: 192.168.1.2
```

Comment:
This is not valid YAML - this file fails to validate.

Also, please create a reference to this file on the generic sonoff page. Otherwise noone will ever find it.